### PR TITLE
Add nmstate k8s job

### PR DESCRIPTION
**Pull request by qinqon**

### CODE REVIEW

**Comment by qinqon** [comment]
/hold
we need to merge https://github.com/nmstate/nmstate/pull/1539 first

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml [hunk 1]
```diff
@@ -106,55 +106,7 @@ presubmits:
         resources:
           requests:
             memory: "8Gi"
-  - name: check-kubevirt-e2e-k8s-1.18
```
Comment (phoracek) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:109]: Looks like a bad revert?

**Comment by qinqon** [comment]
/hold cancel

Dependency merged

**Review by fgimenez — APPROVED** [review]
/lgtm
/approve

**Pull request merged**